_config/config.yml

```diff
@@ -1,14 +1,9 @@
----
-Name: ideannotator
-Before:
-  - 'mysite/*'
-After:
-  - 'framework/*'
-  - 'cms/*'
----
-DevBuildController:
-  extensions:
-    - Annotatable
-DataObjectAnnotator:
-    enabled_modules:
-      - mysite
+---
+Name: ideannotator
+---
+SilverStripe\Dev\DevBuildController:
+  extensions:
+    - Axyr\IDEAnnotator\Annotatable

```

Comment:
Are we switching the namespace to SilverLeague?

Reply:
Sure